­õ4venv/lib/python3.6/site-packages/boto3/exceptions.py 2pyXnh